File: basic/src/main/java/com/yuanshenbin/basic/call/CustomDialogListener.java
package com.yuanshenbin.basic.call;

import android.app.Dialog;
import android.view.View;

/**
 * author : yuanshenbin
 * time   : 2021-02-24
 * desc   :
 */
public interface CustomDialogListener {
    void onCustomLayout(View rootView, Dialog dialog);

    void onOk(Callback callback);

    void onCancel(Callback callback);
}

File: basic-kotlin/src/main/java/com/yuanshenbin/basic/call/CustomDialogListener.kt
package com.yuanshenbin.basic.call

import android.app.Dialog
import android.view.View

/**
 * author : yuanshenbin
 * time   : 2021-02-24
 * desc   :
 */
interface CustomDialogListener<Any> {
    fun onCustomLayout(rootView: View?, dialog: Dialog?)
    fun onOk(callback: Callback<Any>?)
    fun onCancel(callback: Callback<Any>?)
}